app/assets/javascripts/admin/order_cycles/controllers/simple_edit.js.coffee

```diff
@@ -10,7 +10,13 @@ angular.module('admin.orderCycles').controller "AdminSimpleEditOrderCycleCtrl",
     $scope.init()
 
   $scope.$watch 'order_cycle_form.$dirty', (newValue) ->
-      StatusMessage.display 'notice', 'You have unsaved changes' if newValue
+      StatusMessage.display 'notice', t("admin.unsaved_changes") if newValue
+
+  $scope.$watch 'order_cycle_form.$valid', (valid) ->
```

Comment:
Seeing this code duplicated across a number of different controllers, I wonder if it could be extracted to a central location. Perhaps to the `StatusMessage` service, along the lines of `StatusMessage.setValidation(valid)`. What do you think?